data/abilities.ts

```diff
@@ -184,8 +184,21 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 			this.add('-ability', pokemon, 'Unnerve');
 			this.effectState.unnerved = true;
 		},
-		onEnd() {
+		onEnd(pokemon) {
 			this.effectState.unnerved = false;
+			const berries = ['aguavberry', 'figyberry', 'iapapaberry', 'magoberry', 'wikiberry',
+				'sitrusberry', 'oranberry', 'liechiberry', 'ganlonberry', 'salacberry', 'petayaberry', 'apicotberry', 'lansatberry', 'starfberry'];
+			let allyUnnerve = false;
+			for (const ally of pokemon.allies()) {
```

Comment:
I think this would cause problems with FFAs; another opponent could have Unnerve and prevent eating Berries.